drivers/backup/backup.go

```diff
@@ -66,6 +67,9 @@ type Cluster interface {
 	// CreateCluster creates a cluste object
 	CreateCluster(req *api.ClusterCreateRequest) (*api.ClusterCreateResponse, error)
 
+	// CreateNewCluster creates a cluste object
+	CreateNewCluster(name string, cloudCred string, kubeconfigPath string, orgID string) (*api.ClusterCreateResponse, error)
```

Comment:
@raviprasadsn u can suggest better name here. Eventually all APIs we want to replace to accept actual inputs instead of directly request API objects as we discussed the call. This is because logic to construct request API objects is getting duplicated in both torpedo and in backup integration.

Reply:
I agree partly to the need for creating request objects. But this API implementation doesn't just create a request object, it actually invokes the px-backup api to create a cluster.  I believe request creation should be a utility instead of an exported driver interface. Driver interfaces should be uniform and having a structure passed as parameter to interface avoids changing the API itself if for some reason structure of the grpc message changes in future date.

Reply:
@raviprasadsn removed CreateNewCluster() API. keeping just `CreateCluster()` API.